### PR TITLE
Apple Home page Clone

### PR DESCRIPTION
**Pull request by DejazmachMolla**
**Navigation bar**

- The navbar contains the logo, some buttons and a search field

- All of them have a similar gradient effect

- The links take a different gradient on hover than their unhovered state but keep the texture of the unhovered state

**Main Body**

- The main body consists of a stretching Jambo background image followed by a row of 4 blocks with background images

- The Jambo background image has some transparent images and a link on top of it

**Footer**

- The footer is a 2 column grid

- The left column contains two paragraphs that are a little far from their left edge and are still aligned to the left

- The right column contains links separated by a border

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,87 @@
...
+          <li><a href="#">Mac</a></li>
+          <li><a href="#">iPod</a></li>
+          <li><a href="#">iPhone</a></li>
+          <li><a href="#">iPad</a></li>
+          <li><a href="#">iTunes</a></li>
+          <li><a href="#">Support</a></li>
+          <li class="last-list">
+            <input class="search-input" type="text" />
+            <i class="fa fa-search search-icon" aria-hidden="true"></i>
+          </li>
```
Comment (lucasmazo32) [inline index.html]: - [ ] You are missing the element that says "Store".

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes below.

## Required changes:

- [ ] Please enable the stickler for this repo.

### Happy coding! :computer:

## Lucas Mazo

File: index.html (hunk 1 above)
Comment (DejazmachMolla) [reply index.html]: I think the missed element could have been a recommended change as it is not directly related to gradients and backgrounds. Stickler has already been enabled. Just to save time as each code review is taking a long time. Thanks.

File: index.html [hunk 2]
```diff
@@ -1,10 +1,88 @@
...
+  <div class="second-row">
+    <!-- The first column of the lower block -->
+    <div class="col-1"></div>
+    <!-- The second column of the lower block -->
+    <div class="col-2"></div>
+    <!-- The third column of the lower block -->
+    <div class="col-3"></div>
+    <!-- The fourth column of the lower block -->
+    <div class="col-4"></div>
+  </div>
```
Comment (sumancrest0001) [inline index.html]: **Status: Required Changes
Hello @DejazmachMolla,**
You have done an awesome job to fix the issues addressed by the previous TSE and it looks similar to the original one. But it seems that we forgot to mention one requirement in the previous review. Sorry about that.
[Required] These four articles should have links. Please use <a> tag with  div. I know it's a minor change but required one.
[Optional] You can give equal margin to both sides of the footer. currently, it looks asymmetric to me.

**Kindly make the requested change and submit for another review. 
Have a query regarding the code review? DM on slack Suman Shrestha
Happy coding !!**

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Parroved :trophy: :balloon: 

Hi @DejazmachMolla, congratulations on fixing all the changes identified by the previous TSE's :smile: 
The project now meets all the minimum requirement, thanks to your yardwork and the help of our great TSE's :+1: 

You can now merge to master and submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**